tasks/agent5-linux.yml

```
@@ -26,6 +26,21 @@
     enabled: no
   when: not datadog_skip_running_check and not datadog_enabled
 
+- name: Register all checks directories present in datadog
+  find:
+    paths: /etc/dd-agent/conf.d/
+    pattern: "*.yaml"
+    file_type: directory
```

On Agent 5, this directly matches files (as there are no `<check>.d` folders), so the file type should be changed accordingly (otherwise Ansible doesn't match anything since there are no directories ending in `.yaml`)
```suggestion
    file_type: file
```